src/sass/file-upload/file-upload.scss

```diff
@@ -0,0 +1,111 @@
+@import "../variables/variables";
+
+div.file-upload {
+    border-radius: var(--radius-large, 16px);
+
+    &__assets {
+        display: grid;
+        grid-template-columns: repeat(
+            auto-fill,
+            minmax($card-size-default, max-content)
+        );
+        grid-column-gap: var(--spacing-400, 32px);
+        grid-row-gap: var(--spacing-100, 8px);
+
+        &--compact {
+            grid-template-columns: repeat(
+                auto-fill,
+                minmax($card-size-large, max-content)
+            );
+            grid-column-gap: var(--spacing-100, 8px);
+        }
+
+        &--list {
+            grid-template-columns: 1fr;
+            grid-column-gap: var(--spacing-100, 8px);
+        }
+    }
+}
+
+div.file-uploader {
+    border-radius: var(--border-radius-100, 16px);
+
+    &__container {
+        display: flex;
+        flex-direction: row;
+        gap: var(--spacing-300, 24px);
+        cursor: pointer;
```

Comment:
This is a source of contention since we don't have `cursor: pointer` on buttons right now. I think for now you should remove this `cursor` override.

Reply:
Ok I'll remove it for now, it was also raised in the design, so I'll bring it up there